eng/Versions.props

```diff
@@ -76,6 +76,10 @@
     <MicrosoftDotnetToolsetInternalPackageVersion>$(MicrosoftNETSdkPackageVersion)</MicrosoftDotnetToolsetInternalPackageVersion>
     <MicrosoftDotnetTemplateLocatorPackageVersion>$(MicrosoftNETSdkPackageVersion)</MicrosoftDotnetTemplateLocatorPackageVersion>
   </PropertyGroup>
+  <PropertyGroup>
+    <!-- Dependencies from https://github.com/dotnet/roslyn -->
+    <MicrosoftNetCompilersToolsetPackageVersion>4.6.0-3.23259.8</MicrosoftNetCompilersToolsetPackageVersion>
```

Comment:
Should this version match the version in the version.details file: https://github.com/dotnet/installer/blob/9ad370329019ab1e3964813dcb6d2b5740ee9002/eng/Version.Details.xml#LL147C81-L147C81